Stratis.Bitcoin/Features/Consensus/PosConsensusValidator.cs

```diff
@@ -14,17 +14,30 @@ namespace Stratis.Bitcoin.Features.Consensus
     /// TODO
     /// </summary>
     /// <remarks>
-    /// These are the criteria for a new block to be accepted as a valid POS block:
+    /// These are the criteria for a new block to be accepted as a valid POS block at version 3 of the protocol, 
+    /// which has been active since 6 August 2016 07:03:21 (Unix epoch time > 1470467000). All timestamps 
+    /// are Unix epoch timestamps with seconds precision.
     /// <list type="bullet">
-    /// <item>New block's timestamp has to be strictly greater than previous block's timestamp.</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
+    /// <item>New block's timestamp ('BlockTime') MUST be strictly greater than previous block's timestamp.</item>
+    /// <item>Coinbase transaction's (first transaction in the block with no inputs) timestamp MUST be inside interval ['BlockTime' - 15; 'BlockTime'].</item>
+    /// <item>Coinstake transaction's (second transaction in the block with at least one input and at least 2 outputs and first output being empty) timestamp
+    /// MUST be equal to 'BlockTime' and it MUST have lower 4 bits set to 0 (i.e. be divisible by 16) - see <see cref="StakeTimestampMask"/>.</item>
+    /// <item>Block's header 'nBits' field MUST be set to the correct POS target value.</item>
```

Comment:
is it not more correc tot say "the 'nBits' must be bellow the target"

Reply:
hmm, I think nBits defines the target and the coinstake hash must be below that. So I would say it would be incorrect. but am I right?